db/migrate/20220819190425_change_default_user_name.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeDefaultUserName < ActiveRecord::Migration[7.0]
+  def change
+    change_column :users, :name, :string, default: 'Unnamed'
+    User.where(name: 'CHANGEME').update_all(name: 'Unnamed', updated_at: Time.zone.now)
```

Comment:
```suggestion
    User.where(name: 'CHANGEME').update_all(name: 'Name Not Provided', updated_at: Time.zone.now)
```
What do you think about this? It is a bit more clear about what actually happened

Reply:
Changed it!